app.js

```diff
@@ -1,9 +1,14 @@
 var express = require('express');
 var config = require('konfig')();
 var glob = require('glob');
+var redis = require('redis');
+var bodyParser = require('body-parser');
+var cookieParser = require('cookie-parser');
```

Comment:
You need to add body-parser and cookie-parser to the dependencies in package.json